### ISSUE TITLE
Unload Combine changes destination

### ISSUE THREAD

**Issue by Flashsolutions** [Stephan-S/FS19_AutoDrive#945]
AutoDrive Version: 1.0.0.7
Release or custom zip: Release
Map (if applicable): Eiersholt
Vehicle / type (applicable): All Tractors

**Describe the bug**
Unload Combine mode of AutoDrive 1.0.7.0 changes unload destination to target destination after unloading and continue return to field.

The color of the destination and target also change from white to another color. See youtube video for full explanation and settings I am using. 

**If applicable - Steps to reproduce the behavior**
See youtube video https://youtu.be/xVzKIKaqTxU and screenshot

![fsScreen_2019_11_25_09_49_44](https://user-images.githubusercontent.com/60280/69551591-f0da2680-0f6a-11ea-9b79-d4e49f062f8e.png)

**Screenshots**

**Comment by Stephan-S** [comment]
@Iwan1803 You can stop adding the 'further-investigations' label to every issue. It becomes meaningless. I don't even know if that means you are doing some investigations or if this is supposed to mean I am supposed to investigate...

@Flashsolutions 
Good idea to show the settings in the video :-) You have the multiple targets option enabled. This means he will use the next target in the same folder for his next run. And this appears to be the 'Field' destination in this case.
So just disable 'Multiple targets'

**Comment by Flashsolutions** [comment]
Interesting.  I assumed multiple targets meant there were multiple tractors seeking to unload the combine.  So it sounds like your description is misleading (Multiple Destinations)?  I was trying to setup the scenario  where two tractor trailers serviced one combine.  If I set Multiple Targets to 'No',  both tractors would return to the same waypoint to wait for the combine to fill.  When I set multiple targets to 'yes', they waited in the field and thus did not run into conflicts until they finished loading and the destination changed to the field.  Is there a combination of settings I should use to accomplish my goal?
